packages/extension-redis/src/Redis.ts

```diff
@@ -134,7 +134,9 @@ export class Redis implements Extension {
     }
     this.sub.on('messageBuffer', this.handleIncomingMessage)
 
-    this.redlock = new Redlock([this.pub])
+    this.redlock = new Redlock([this.pub], {
+      retryCount: 0,
```

Comment:
not sure why we havent had this, I actually thought 'lockTimeout' would trigger an error if it can't lock after X ; However, what we're doing is just silently ignoring the lock after X...